homeassistant/components/netatmo/climate.py

```diff
@@ -82,90 +148,275 @@ def temperature_unit(self):
     @property
     def current_temperature(self):
         """Return the current temperature."""
-        return self._data.current_temperature
+        return self._data.room_status[self._room_id]['current_temperature']
 
     @property
     def target_temperature(self):
         """Return the temperature we try to reach."""
-        return self._target_temperature
+        return self._data.room_status[self._room_id]['target_temperature']
 
     @property
     def current_operation(self):
         """Return the current state of the thermostat."""
-        state = self._data.thermostatdata.relay_cmd
-        if state == 0:
+        state = self._data.room_status[self._room_id]['heating_status']
+        if state is False:
```

Comment:
Is it possible that state can be anything different than false or true?

Reply:
In normal situations, it should be true or false, but if the room does not contain any thermostat or valve, there might be a rare chance to have this value as `None`. This is just to avoid such case.